hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/AbstractS3AMockTest.java

```diff
@@ -58,7 +60,12 @@ public void setup() throws Exception {
     Configuration conf = createConfiguration();
     fs = new S3AFileSystem();
     URI uri = URI.create(FS_S3A + "://" + BUCKET);
-    fs.initialize(uri, conf);
+    try {
```

Comment:
as this is a mock test, best to just remove the s3guard and cse settings from the config we've created. No real FS should be involved.
We already set metastore to null in `createConfiguration()`.

Reply:
`TestObjectChangeDetectionAttributes`, overrides `createConfiguration()` to set metastore, so we should just skip, right?

Reply:
better to just remove the CSE settings. This is mock, there's no communication to S3 at all, so no encryption taking place.